README.md

```diff
@@ -2,6 +2,25 @@
 
 This library provides a programmatic way to manage Datadog synthetic browser tests. This library contains all the necessary functions that are required to manage the synthetic browser tests with different steps and configurations.
 
+[![License](https://img.shields.io/badge/License-MIT-brightgreen)](https://github.com/personio/datadog-synthetic-test-support/blob/master/LICENSE)
+[![Version](https://img.shields.io/github/v/release/personio/datadog-synthetic-test-support)](https://github.com/personio/datadog-synthetic-test-support/releases)
+[![Release date](https://img.shields.io/github/release-date/DataDog/datadog-api-client-java)](https://github.com/personio/datadog-synthetic-test-support/releases)
+![Downloads](https://img.shields.io/github/downloads/personio/datadog-synthetic-test-support/total)
```

Comment:
The download does not have a URL for it. Is that correct? Will there be an URL at some point?